frontend/src/views/Pages/Register.vue

```diff
@@ -183,6 +183,7 @@ export default {
             firstName: this.form.firstname,
             lastName: this.form.lastname,
             password: this.form.password.password,
+            language: this.$store.state.language,
```

Comment:
line 60 of store.js:
`language: null`

Reply:
I think it is since we arrive on the login where we have to choose between german and english and then we have something in it.
But I will check if their a possibility where it could brake!

Reply:
Yes, when we are not logged in, we have no locale in the store. During register we are not logged in.